packages/samples/common-types/src/mobo.tsp

```diff
@@ -6,4 +6,4 @@ using OpenAPI;
 
 namespace Azure.ResourceManager.CommonTypes;
 
-@@extension(ManagedOnBehalfOfConfiguration.moboBrokerResources, "x-ms-identifiers", ["id"]);
+@@extension(ManagedOnBehalfOfConfiguration.moboBrokerResources, "x-ms-identifiers", #["id"]);
```

Comment:
```suggestion
@@identifiers(ManagedOnBehalfOfConfiguration.moboBrokerResources, #["id"]);
```

I think would be the correct new way

Reply:
Talked offline - there's still some work to do to get `@identifiers` working everywhere. Using it here causes some 'duplicate' errors once ARM is imported.